bin/ch-run.c

```diff
@@ -216,6 +216,12 @@ int main(int argc, char *argv[])
    case IMG_NONE:
       FATAL("unknown image type: %s", args.c.img_ref);
       break;
+   case OCI_BUNDLE:
+      if (args.c.newroot != NULL)  // --mount was set
+         WARNING("--mount invalid with directory image, ignoring");
+      args.c.newroot = realpath_(cat(args.c.img_ref, "/rootfs"), false);
+      img_directory_verify(args.c.newroot, &args);
```

Comment:
Three of the four lines here are identical to `IMG_DIRECTORY`, which isn’t DRY. Is there a way to re-work things to avoid repeated code?